<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gams::studio::debugger::DebugWidget</class>
 <widget class="QWidget" name="gams::studio::debugger::DebugWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>22</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="tbRun">
     <property name="toolTip">
      <string>Continue running</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../icons/icons.qrc">
       <normaloff>:/solid/play</normaloff>:/solid/play</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tbStep">
     <property name="toolTip">
      <string>Step to the next possible line</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../icons/icons.qrc">
       <normaloff>:/solid/step</normaloff>:/solid/step</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tbPause">
     <property name="toolTip">
      <string>Pause at the next possible line</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../icons/icons.qrc">
       <normaloff>:/solid/pause</normaloff>:/solid/pause</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tbStop">
     <property name="toolTip">
      <string>Stops the execution</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../icons/icons.qrc">
       <normaloff>:/solid/stop</normaloff>:/solid/stop</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>8</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>8</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::VLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="laInfo">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
